# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/RemoteAccessibleShared.h
# Commit: ac04b727a98b
# Full Hash: ac04b727a98bb1d4e0644665b59ed4da3e9e3858
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P8: Move GetGroupPosition to Accessible. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D134211
# ==============================================================================

diff -r 5d76535b47fc -r ac04b727a98b accessible/ipc/RemoteAccessibleShared.h
--- a/accessible/ipc/RemoteAccessibleShared.h	Tue Jan 04 21:01:37 2022 +0000
+++ b/accessible/ipc/RemoteAccessibleShared.h	Tue Jan 04 21:01:37 2022 +0000
@@ -65,7 +65,7 @@
 
 nsStaticAtom* ARIARoleAtom() const;
 
-mozilla::a11y::GroupPos GroupPosition();
+virtual mozilla::a11y::GroupPos GroupPosition() override;
 void ScrollTo(uint32_t aScrollType);
 void ScrollToPoint(uint32_t aScrollType, int32_t aX, int32_t aY);
 
